Other email views

PatricKangasmaki opened the pull request.

PatricKangasmaki commented:
Adds emails:
 - Project invitation
 - Email verification

Follows the pattern created by the pull request: Forgot password email #271 and should be reviewed only after that one has been approved.

PatricKangasmaki commented:
Removed test routes and rebased on master.

juhoeerola commented on frontend/src/components/emails/ProjectInvitationEmail.tsx:
```diff
@@ -0,0 +1,62 @@
+        </div>
+      </div>
+      <div style={{ ...textStyle, marginTop: '40px', marginBottom: '-40px' }}>
+        <i>Don’t want to join the project? You can ignore this message..</i>
```
```suggestion
        <i>Don’t want to join the project? You can ignore this message.</i>
```

PatricKangasmaki commented:
> This could be moved to e.g. the EmailTemplate module as a higher order component, as it starts to be duplicated quite a few times.

Great point, added that. I renamed htmlBox and improved its' styling a bit as well. And also merged the other PR here which had task rating notification emails, I felt like it didn't really need its own branch when it is basically doing the same thing as these other email views, just different texts

juhoeerola reviewed (approved):
👍
